### PR TITLE
BD-2548: Updating Content Calendar docs for new UI

### PR DESCRIPTION
**Pull request by internetisaiah**
# Pull Request/Issue Resolution

#### Description of Change:
I'm updating the [Content Calendar doc](https://www.braze.com/docs/user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/campaign_calendar/) due to the new UI updates.

- Jira Ticket: [BD-2548](https://jira.braze.com/browse/BD-2548)

**Notes for reviewer:**

- I was a bit unsure of how to properly format/capitalize words like "Send Calendar". Not sure if it's a feature/product name and requires title-case or its merely a UI element that uses sentence-case for headings and lower-case for inline references.
- Not totally sure the best way to do sections _after_ the intro. Personally, I think I could have gone from the intro directly into `To open the calendar, select Messaging > Content Calendar.` However, the original doc had more information there.

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @bre-fitzgerald as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Maria Maldonado on the Legal team has signed off.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @bre-fitzgerald for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure) (tag @rachel-feinberg for Liquid use cases)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/campaign_calendar.md [hunk 1]
```diff
@@ -5,42 +5,43 @@ page_order: 4
...
-{% endalert %}
+Messaging Analytics displays messaging data for your recent and upcoming scheduled campaigns. You can view the messaging data for a specific day by hovering over that day in either graph:
+
+- **Upcoming Message Types:** The number of push, email, and in-app messaging campaigns scheduled within the next 14 days.
+- **Messages Sent:** The number of messages sent and opened within past 14 days.
+
+![The 'Upcoming Message Types' and 'Messages Sent' graphs within Messaging Analytics.]({% image_buster /assets/img/campaign_calendar/content_calendar_messaging_analytics.png %})
 
-### Future send calendar
+To download a copy of your data, select ![Hamburger menu.][1]{: style="border:0;"} **Chart context menu**, then choose your preferred file format.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/campaign_calendar.md:33]: Ahhhh I see your confusion from earlier. I should've been clearer, my mistake. You don't need to download these icons, we have the fontawesome library installed in the repo so you can reference them with the HTML embed.

```suggestion
To download a copy of your data, select <i class="fa-solid fa-bars" style="color: #2e7487;"></i> **Chart context menu**, then choose your preferred file format.
```

File: _docs/_user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/campaign_calendar.md [hunk 2]
```diff
@@ -5,42 +5,43 @@ page_order: 4
...
 
-**Red:** Single-channel scheduled campaigns<br>
-**Blue:** Multichannel scheduled campaigns
+## Send Calendar
 
-Note that only **Scheduled Campaigns** are displayed in the Campaign Calendar. Canvas steps, triggered campaigns, and API campaigns will not be displayed here. 
+The Send Calendar displays all your scheduled campaigns for the current month. You can change the month by selecting ![Chevron left.][2]{: style="border:0;max-width: 20px;"} **See previous month** or ![Chevron right.][3]{: style="border:0;max-width: 20px;"} **See next month**. To return to the current month, select **Today**.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/campaign_calendar.md:37]:
```suggestion
The Send Calendar displays all your scheduled campaigns for the current month. You can change the month by selecting <i class="fa-solid fa-chevron-left" style="color: #2e7487;"></i> **See previous month** or <i class="fa-solid fa-chevron-right" style="color: #2e7487;"></i>**See next month**. To return to the current month, select **Today**.
```

File: _docs/_user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/campaign_calendar.md [hunk 3]
```diff
@@ -5,42 +5,43 @@ page_order: 4
...
+![The Send Calendar displaying all scheduled campaigns in the current month.]({% image_buster /assets/img/campaign_calendar/content_calendar_sends.png %})
 
-**Upcoming Message Types:** Details the amount of push, email, and in-app messaging campaigns over the course of the next 14 days.<br> 
-**Messages Sent:** Details the number of messages sent and opened over the course of the past 14 days. 
+Each color on the calendar represents a different scheduled campaign type:
 
-To get more detailed information on these analytics, you can scroll over the individual bars within the charts for a breakdown by day. If you wish to save the message analytics charts, you can press the ![Hamburger menu.][3]{: style="border:0;"} icon to save the chart in various formats. 
+- **Red:** Single-channel scheduled campaigns.
+- **Blue:** Multichannel scheduled campaigns.
+- **Gold:**
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/campaign_calendar.md:45]: Let me find out what gold is, good catch

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]

**Comment by bre-fitzgerald** [comment]
> I was a bit unsure of how to properly format/capitalize words like "Send Calendar". Not sure if it's a feature/product name and requires title-case or its merely a UI element that uses sentence-case for headings and lower-case for inline references.

Our approach with stuff like that has been:
- If you are referring to an interaction with the UI or directly to a UI element, format it exactly as it appears on screen and bold it.
- If you are referring to it as a general concept, do not capitalize it unless it is a branded feature. Branded features should be all listed in our glossary in the style guide, but you can also refer to the [marketing glossary](https://confluence.braze.com/display/MAR/Braze+Glossary#BrazeGlossary).

@lydia-xie @rachel-feinberg did I get that right?

File: _docs/_user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/campaign_calendar.md [hunk 4]
```diff
@@ -5,42 +5,43 @@ page_order: 4
...
-![][5]{: style="float:right;max-width:15%;margin-left:15px;margin-bottom:10px;margin-top:10px;"}
+> The Content Calendar offers a high-level overview for your upcoming scheduled campaigns.
 
-To access the calendar, go to **Messaging** > **Content Calendar**.
+You can analyze messaging data for upcoming and previous scheduled messages, as well as view and manage your scheduled campaigns for the month. To open the calendar, select **Messaging** > **Content Calendar**.
 
 {% alert note %}
-If you are using the [older navigation]({{site.baseurl}}/navigation), this page is located at **Engagement** > **Campaigns** > **Calendar**.
+If you're using the [older navigation]({{site.baseurl}}/navigation), select **Engagement** > **Campaigns** > **Calendar**.
 {% endalert %}
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/campaign_calendar.md:23]: > Not totally sure the best way to do sections after the intro.

Hmm, could always do a brief lead-in sentence to give context to the next two sections. Something like: 

The content calendar contains two main sections:
- Messaging Analytics
- Send Calendar

> Personally, I think I could have gone from the intro directly into `To open the calendar, select Messaging > Content Calendar`. However, the original doc had more information there.

What you have seems good to me! It looks like you distributed that information throughout the doc, which seems like the right move to me.

File: _docs/_user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/campaign_calendar.md (hunk 3 above)
Comment (internetisaiah) [reply _docs/_user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/campaign_calendar.md]: All other fixes have been committed. Waiting on a response from product [here](https://brazetechnology.slack.com/archives/C03NCGX19KQ/p1698265899401969).

**Review by bre-fitzgerald — APPROVED** [review]
Looks great!! Thanks for tackling this 💪

**Pull request merged**